api/api/V1/Processing.ts

```diff
@@ -352,10 +367,193 @@ export default function (app: Application, baseUrl: string) {
     },
   );
 
+  /**
+   * @api {post} /api/v1/:id/tracksAndTags Add tracks and tags to a recording
+   * @apiName PostTracksAndTags
+   * @apiGroup Processing
+   *
+   * Requires super-admin user credentials
+   *
+   * @apiParam {JSON} data Data which defines the tracks and tags (type specific).
+   * @apiParam {Number} AlgorithmId Database ID of the Tracking algorithm details retrieved from
+   * (#FileProcessing:Algorithm) request
+   *
+   * @apiUse V1ResponseSuccess
+   * @apiSuccess {int[]} trackIds of the newly created track.
+   *
+   * @apiUse V1ResponseError
+   *
+   */
+  app.post(
+    `${apiUrl}/:id/tracksAndTags`,
```

Comment:
Can we keep api endpoints with the consistent kebab-case style that's generally seen as being more standard for urls?  `tracksAndTags` -> `tracks-and-tags`